docs/paas-plus/dev/troubleshooting/decline-changes.md

```diff
@@ -0,0 +1,16 @@
+---
+title: How to decline the Spryker Code Upgrader changes
```

Comment:
What would be the error if I tried to run the upgrader when there was an open PR?